### PR TITLE
Update UserGuide for Countdown feature

### LINKED ISSUE #109: Update User Guide - COUNTDOWN feature

### PR DESCRIPTION
**Pull request by cwenling**
subfeatures:
- Switching to Countdown tab
- Adding a new event
- Adding an event from task list
- Deleting an event

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -284,6 +284,85 @@ Format: `clear`
 #### Modules tab features
 
 #### Countdown tab features
+You can countdown to your most essential upcoming events in the Countdown tab. You can know at a quick glance, how many
+you have left to that Final Exam or that birthday party.
+
+##### Switching to Countdown tab: `countdown`
+To switch to the Countdown tab, simply type `planner` into the command line. You can also click on the Countdown button
```
Comment (ruilingk) [inline docs/UserGuide.md]: think you meant `countdown`! haha

File: docs/UserGuide.md [hunk 2]
```diff
@@ -284,6 +284,85 @@ Format: `clear`
...
+![switch tab](images/countdowntab_switchtab.png)
+Figure x: TR4CKER after executing `countdown`
+
+The `Upcoming Events` panel shows the list of upcoming events that are counted down to.
+
+Example:
+* `countdown` - Switches to Countdown tab, showing the list of events.
+
+##### Adding a new event to countdowns list
+You can add an event to the list of events, and TR4CKER help you to countdown to it.
```
Comment (ruilingk) [inline docs/UserGuide.md]: and TR4CKER **helps** you to...?

File: docs/UserGuide.md [hunk 3]
```diff
@@ -284,6 +284,85 @@ Format: `clear`
...
+
+Format: `countdown n/NAME d/DATE`
+
+* Adds an event with name `NAME` and date `DATE` to the Countdown events list.
+* Both fields are compulsory and can be in any order.
+* `NAME` must be in the alphanumeric format (only containing letters and numbers) and preferably under 
+100 characters long.
```
Comment (ruilingk) [inline docs/UserGuide.md]: what if the user exceeds 100 characters?

**Review by ruilingk — COMMENTED** [review]
some minor changes and it lgtm!

**Review by hanweic53 — APPROVED** [review]
Looks good to merge.

**Pull request merged**